requirements-dev.txt

```diff
@@ -1,6 +1 @@
-django-debug-toolbar
-django-webtest
-factory-boy
-coverage
-ipython
-ipdb
```

Comment:
I'm not opposed to leaving IPython and ipdb out of the dev dependencies (though I find them helpful!), I'm just curious if you had a specific reason for omitting them.